docs/data-sources/cce_nodes.md

```diff
@@ -0,0 +1,94 @@
+---
+subcategory: "Cloud Container Engine (CCE)"
+---
+
+# huaweicloud_cce_nodes
+
+Use this data source to get a list of CCE nodes.
+
+## Example Usage
+
+```hcl
+variable "cluster_id" {}
+variable "node_name" {}
+
+data "huaweicloud_cce_nodes" "node" {
+  cluster_id = var.cluster_id
+  name       = var.node_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to obtain the cce nodes. If omitted, the provider-level
```

Comment:
`cce` is a short name, uppercase is recommended

Reply:
fixed